integration-libs/s4-service/order/components/s4-service-order-detail-actions/s4-service-order-detail-actions.component.html

```diff
@@ -1,5 +1,5 @@
 <ng-container *ngIf="order$ | async as order">
-  <div class="cx-order-details-actions row" *ngIf="displayActions$">
+  <div class="cx-order-details-actions row" *ngIf="checkServiceStatus(order)">
```

Comment:
```suggestion
  <div class="cx-order-details-actions row" *ngIf="displayActions$ | async">
```
Probably the problem you mentioned in the private message was related to fact that `async` pipe was missing in this place. The condition checked only whether the observable is defined, but not its value.

Reply:
Thanks for pointing out.